### ISSUE TITLE
Specify mongo database name in the configuration file.

### ISSUE THREAD

**Issue by warownia1** [warownia1/Slivka#77]
Since multiple separate Slivka instances may use the same database, it's necessary to specify the database name for each project individually instead of hard-coded "slivkadb".

Options include:
- Full %-encoded mongodb URI: `mongodb://[username:password@]host[:port][/database]`
  Very strict and unambiguous, but hard to use.
- Mongodb URI but without %-encoding `mongodb:\/\/(?:([%\w]+):(.*)@)?([\w./]+(?::\d+)?)\/(\w+)`
  Easier to use, but may produce ambiguous results if uname or password contains special characters.
- Passing connection data object e.g.:
  ```
  host: 127.0.0.1:8000
  socket: /tmp/mongo.sock
  username: warownia1
  password: P@44w0rd
  database: slivkadb
  ```

**Comment by warownia1** [comment]
If passed a string, strip database name and pass the rest directly to the `pymongo.MongoClient`
```
host, dbname = hostname.rstrip('/', 1)
database = MongoClient(host)[dbname]
```

**Comment by warownia1** [comment]
Closed by 2fca2b198354a5de66f3cfddafe41247054d712f

**Comment by warownia1** [comment]
Closed by e0ff034758b55642a371b5c20e60059c7cec1ed3
